SDWebImage/SDImageCache.m

```diff
@@ -287,6 +289,21 @@ - (void)diskImageExistsWithKey:(nullable NSString *)key completion:(nullable SDW
     });
 }
 
+- (BOOL)diskImageDataExistsWithKey:(nullable NSString *)key {
```

Comment:
This seems no IO-safe(not be called on IO Queue). Maybe we should rename this method to `safeDiskImageDataExistsWithKey:`, and then sync/async public API call this method with `dispatch_sync(self.ioQueue)` and `dispatch_async(self.ioQueue)`